provisioning/miniprov/hare_mp/main.py

```diff
@@ -445,10 +445,36 @@ def start_mkfs_parallel(hostname: str, hare_config_dir: str):
                       f'{perf_result}\n\n')
 
 
+@repeat_if_fails()
+def wait_for_mkfs_to_complete_on_all_nodes(utils: Utils,
```

Comment:
Can it be shorter a bit?

```suggestion
def wait_mkfs_complete_on_all_nodes(utils: Utils,
```

Reply:
Okay.